src/main/java/com/google/step/snippet/servlets/SearchServlet.java

```diff
@@ -91,7 +92,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     String stackLink = getLink(STACK_CSE_ID, query);
     if (stackLink != null) {
       // allLinks.add(stackLink);
-      StackOverflowClient stackClient = new StackOverflowClient();
+      Client stackClient = new StackOverflowClient();
```

Comment:
With the client we can implement a for loop now per Joan's suggestion in #19 [comment](https://github.com/googleinterns/step1-2020/pull/19#discussion_r453908486) ?

Also can address #30 in this PR?

Reply:
i think it solved #19 comment but i don't quite understand what #30 is referring to.

Reply:
Nothing too complicated. #30 says that Card.java is currently declaring its members as `private String title` but it could be `private final String title`, etc.